### PR TITLE
[engraving] Added tremolo two and single chord. Step 1

### PR DESCRIPTION
**Pull request by igorkorsukov**

### CODE REVIEW

File: src/engraving/tests/compat114_tests.cpp [hunk 1]
```diff
@@ -70,7 +70,7 @@ TEST_F(Engraving_Compat114Tests, articulations) {
 }
 
 TEST_F(Engraving_Compat114Tests, chord_symbol) {
-    compat("chord_symbol");
+    //compat("chord_symbol");
```
Comment (cbjeukendrup) [inline src/engraving/tests/compat114_tests.cpp:73]: Why is this test disabled? Wouldn't it be better to fix it?

Comment (igorkorsukov) [reply src/engraving/tests/compat114_tests.cpp]: Disabled locally (they fail locally for me)
Turned it on.. thanks!

**Review by cbjeukendrup — APPROVED** [review]

**Comment by cbjeukendrup** [comment]
(About the "dispatcher" pattern, I'm not sure whether this will work perfectly, because theoretically there may be some virtual methods of EngravingItem that the original Tremolo didn't override but that the dispatcher does need to override to dispatch them to the new tremolo classes. But as a very temporary solution, I think it's fine.)

**Comment by igorkorsukov** [comment]
> (About the "dispatcher" pattern, I'm not sure whether this will work perfectly, because theoretically there may be some virtual methods of EngravingItem that the original Tremolo didn't override but that the dispatcher does need to override to dispatch them to the new tremolo classes. But as a very temporary solution, I think it's fine.)

Yes, this is just very temporary solution )

**Pull request merged**